File: app/projects/domain/src/lib/quiz-run-steps/answer/answer-step.ts
import { MessageStatus } from '@domain/infrastructure/message-status';
import { Subscription } from '../../subscriptions/subscription';

export interface AnswerStepEvent {
  answer: AnswerStep;
  status: MessageStatus;
  message: string;
}

export interface AnswerStep {
  quizId: number;
  questionId: number;
  quizRunStepId: number;
  subscription: Subscription;
}

export interface SendAnswer {
  quizId: number;
  quizRunStepId: number;
  subscriptionId: number;
  answer: string;
}

File: service/Api/QuizRunStep/Answer/AnswerStep.cs
namespace Quizmeister
{
    public class AnswerStep
    {
        public int QuizId { get; set; }
        public int QuizRunStepId { get; set; }
        public int QuestionId { get; set; }
        public Subscription Subscription { get; set; }
    }
}
